### PR TITLE
Cleanup tpm*.permall before creating sock file

### PR DESCRIPTION
**Pull request by Amrysliu**
Related: https://progress.opensuse.org/issues/107155

### CODE REVIEW

File: backend/qemu.pm [hunk 1]
```diff
@@ -523,6 +523,13 @@ sub setup_tpm ($self, $arch) {
     mkdir $vmpath unless -d $vmpath;
     my $vmsock = "$vmpath/swtpm-sock";
     unless (-e $vmsock) {
+        # Before create swtpm-sock, we should make sure there is no tpm*.permall file
+        # When tpm version is 2.0, the file is tpm2-00.permall.
+        # When tpm version is 1.x, the file is tpm-00.permall.
+        # See: https://progress.opensuse.org/issues/107155
+        runcmd(('rm', "$vmpath/tpm-00.permall")) if (-e "$vmpath/tpm-00.permall");
```
Comment (rfan1) [inline backend/qemu.pm:530]: CI check failed.

at the same time, how about using "rm -rf tpm*.permall"?

Comment (coolo) [reply backend/qemu.pm:530]: this is perl not bash

Comment (perlpunk) [reply backend/qemu.pm:530]: e.g.
```
unlink glob "tpm*.permall";
```

Comment (Amrysliu) [reply backend/qemu.pm]: Modified. Thanks all.

**Review by Martchus — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**